include/WSS.hpp

```diff
@@ -0,0 +1,87 @@
+#pragma once
+
+#include <EVT/io/CANDevice.hpp>
+#include <EVT/io/CANOpenMacros.hpp>
+#include <EVT/io/SPI.hpp>
+#include <co_core.h>
+#include <dev/hallSensor.hpp>
+
+#define NUM_HALLSENSORS 2
+
+using namespace EVT::core::IO;
+
+namespace WSS {
+
+/**
+ * This is an example of a class for a board
+ */
+class WSS : public CANDevice {
+public:
+    WSS(DEV::hallSensor& hallSensor1, DEV::hallSensor& hallSensor2);
+
+    static constexpr uint16_t NODE_ID = 8;
+
+    /**
+     * Gets the object dictionary
+     *
+     * @return an object dictionary
+     */
+    CO_OBJ_T* getObjectDictionary();
+
+    /**
+     * Updates the temperature values in an array and updates the error array from the TMU object.
+     */
+    void process();
+
+    /**
+     * Get the device's node ID
+     *
+     * @return The node ID of the can device.
+     */
+    uint8_t getNodeID() override;
+
+    /**
+     * Get the number of elements in the object dictionary.
+     *
+     * @return The number of elements in the object dictionary
+     */
+    uint8_t getNumElements() override;
+
+private:
+    DEV::hallSensor* hallSensors[NUM_HALLSENSORS]{};
+
+    uint32_t timeDifference[NUM_HALLSENSORS] = {0, 0};
```

Comment:
Make both of these variables plural
```suggestion
    uint32_t timeDifferences[NUM_HALLSENSORS] = {0, 0};
```